webrtc-stats.html

```diff
@@ -768,6 +768,7 @@
              double             burstDiscardRate;
              double             gapLossRate;
              double             gapDiscardRate;
+             DOMHighResTimeStamp  lastPacketReceivedTimestamp;
```

Comment:
> I always thought that the timestamp in the getStats() for local objects was timestamp when the snapshot of the getStats was created, which may have been when the getStats was called.

Thanks, I got confused. That sounds right (ironically, [remoteTimestamp](https://w3c.github.io/webrtc-stats/#dom-rtcoutboundrtpstreamstats-remotetimestamp) contains a clearer definition of the local timestamp than timestamp does: *"This differs from [timestamp](https://w3c.github.io/webrtc-pc/#dom-rtcstats-timestamp), which represents the time at which the statistics were generated or received by the local endpoint."*)

So in remote reports, `lastPacketReceivedTimestamp` is in the clock of the remote?